### PR TITLE
Improve contributing experience (esp on windows)

### PR DESCRIPTION
**Pull request by OlegYch**
- mention publishBridges in guide
- update protoc plugin to avoid using python on windows
- update whitesource plugin to avoid using missing pecoff dependency (which breaks idea import)
- ignore idea generated files
- fix tests on windows

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=1.x&zinc=pull/587/head&io=1.x&librarymanagement=1.x&util=1.x&website=1.x&deploy_to_bintray=false&run_sbt_community=false&verbose_dbuild=false&community_verbose=false">this link.</A>

File: project/plugins.sbt [hunk 1]
```diff
@@ -2,7 +2,7 @@ addSbtPlugin("org.scala-sbt" % "sbt-houserules" % "0.3.7")
 addSbtPlugin("org.scala-sbt" % "sbt-contraband" % "0.4.0")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.3.3")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "3.0.2")
-addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.12-rc5")
-libraryDependencies += "com.trueaccord.scalapb" %% "compilerplugin" % "0.6.0"
-addSbtPlugin("com.lightbend" % "sbt-whitesource" % "0.1.9")
+addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.15")
```
Comment (jvican) [inline project/plugins.sbt]: Unfortunately we cannot do this, see https://github.com/sbt/zinc/issues/546

**Review by jvican — CHANGES_REQUESTED** [review]
CI is failing, otherwise LGTM

File: project/plugins.sbt (hunk 1 above)
Comment (OlegYch) [reply project/plugins.sbt]: ok reverted

**Review by jvican — APPROVED** [review]
LGTM! Thank you

**Review by eed3si9n — APPROVED** [review]

**Pull request merged**